src/completions.js

```diff
@@ -36,7 +36,27 @@ function activate(context) {
         }
     });
 
-    context.subscriptions.push(kw_provider);
+    const sn_provider = vscode.languages.registerCompletionItemProvider("graphene", {
+        provideCompletionItems(document, position, token, context) {
+            const for_sn = new vscode.CompletionItem(
+                { label: "for", description: "Code snippet for for loop" }
+            );
+            for_sn.insertText = new vscode.SnippetString(
+                "for ${1:i} in ${2:iterator} {\n\t${0:$BLOCK_COMMENT_START code $BLOCK_COMMENT_END}\n}"
+            );
+
+            const for_range_sn = new vscode.CompletionItem(
```

Comment:
This line requires `@require_one "std/iterators.c3"`... Is it possible to also add the import when we use the snippet (provided it's not already listed as an import)?

Reply:
I haven't been able to find the right API for that yet, but it'd definitely be nice

Reply:
Done! Although it could be improved; it simply adds the import after the last import in the file, or at the start of the file if there are no other imports.